### ISSUE TITLE
Allow users to add fields "on the fly"

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1758]
### Expected Behavior
Allow users to add fields (presumably to the JSON blob field) "on the fly" in the program as needed, without importing a file to create those fields.
### Actual Behavior 
n/a
### Steps to Reproduce
n/a
### Instance Information
n/a

**Comment by RDmitchell** [comment]
@nllong / @macintoshpie -- having this functionality came up in a call with Washington DC -- they would really like this ability.

**Comment by macintoshpie** [comment]
Interesting, do you know their use-case? Are there just one or two properties that are missing a column for some reason?

**Comment by RDmitchell** [comment]
@nllong -- do you remember if Andrew Held mentioned what they need it for? 

I can email him to get more specifics if that is useful.

**Comment by nllong** [comment]
They want to add a column such as "User Verified" and have the user edit it on the Inventory Detail page and update the field from a 0 to a 1. I imagine they would then use a DQ rule to add a label to the properties that haven't been verified.

**Comment by macintoshpie** [comment]
Not sure when we'll pick this up, but just to capture some thoughts
- you should only be able to create new columns from the Organizations Column Settings page
- it should be an extra_data column

**Comment by RDmitchell** [comment]
@macintoshpie -- yes, those are what we were thinking

**Comment by RDmitchell** [comment]
I think this issue is still relevant and should be kept open.

**Comment by isalanglois** [comment]
@RDmitchell  I'll move it to the new board

**Comment by RDmitchell** [comment]
see #3780 
Column Settings: getting 520 error when making a new column but column is created

**Comment by RDmitchell** [comment]
@haneslinger / @isalanglois -- see the bug above that I found when testing last week.

**Comment by RDmitchell** [comment]
@haneslinger / @axelstudios -- is this bug still a problem? The related issue is in test and maybe it shouldn't be?

**Comment by RDmitchell** [comment]
See the comments I just added to #3780

**Comment by RDmitchell** [comment]
Depending on the answer to #3780, I will close both of these issues.

**Comment by RDmitchell** [comment]
I am repeating the comment I made in #3780 

Instance: LBNL staging
SHA: 992862f09
Release: 2.17.1
Org: LBNL Test 200

When I added the field in Column Settings, this error was displayed in Developer tools.
![image](https://user-images.githubusercontent.com/6314950/215232872-1041918f-ce1d-4ddd-af76-039e7eb5d170.png)

However, the program finished adding the field (the spinner stopped) and it was displayed in the Column Settings List
![image](https://user-images.githubusercontent.com/6314950/215232976-e35ce800-5e65-47a6-8ff6-9480d3fc4863.png)

I could edit the field and add data to it in the Property Details view, and it displays properly in the Property List view
![image](https://user-images.githubusercontent.com/6314950/215233028-db0ebfb2-94b8-4e10-a72c-70bd2b8dbb4e.png)

@haneslinger -- do we need to worry about that error? It's not something the user sees, and the program finished making the field. ??
